src/api/resolvers/Query.js

```diff
@@ -1,9 +1,28 @@
 const Query = {
-
   projects: async (_, { _ids }, context) => {
     return await context.models.Project.getProjects(_ids);
   },
 
+  batches: async (_, { input }, context) => {
+    const response = await context.models.Batch.queryByFilter(input);
+    const { previous, hasPrevious, next, hasNext, results } = response;
+    return {  // reurn ImagesConnection
+      pageInfo: {
+        previous,
+        hasPrevious,
+        next,
+        hasNext
+      },
+      batches: results
+    };
+  },
+
+  batch: async (_, { _id }, context) => {
+    return await context.models.Batch.queryById(_id, {
+      remaining: true
```

Comment:
@nathanielrindlaub I left this in here for optimization reasons - there are several codepaths that require the Batch object from the DB but don't require the SQS data. This speeds up these codepaths by letting them access the Batch object with a single DB call instead of a single DB call  + multiple API calls to AWS.

Reply:
Gotcha. Thanks for the clarification!